resources/views/emails/exposure.blade.php

```diff
@@ -0,0 +1,13 @@
+@component('mail::message')
+# Hello {{ $first_name }},
+
+You are receiving this message because you visited the Student Competition Center {{ $date }} according to the People Counter system.
```

Comment:
```suggestion
You are receiving this message because you visited the Student Competition Center {{ $date }}. according to the People Counter system.
```